### DISCUSSION TITLE
Annotations and Figures

### DISCUSSION THREAD

**Discussion by ewolfe1** [thegetty/quire#946] [Get Help] [answered]
I'm very new to Quire, so still learning how it all works, but I am trying to do something that doesn't seem to be working. I suspect it is by design, but maybe there is a workaround?

Basically, I am trying to display a multipage object. I would like to use the 'annotations' / 'input: radio' to be able to allow the user to select different pages. This would minimize display space on the page (i.e., show thumbnail of page one, but allow access to the entire object). 

However, all of the images are resized to fit the dimensions of 'page 1', making all of the subsequent pages squashed/stretched. Again, since these are "annotations", I imagine that this is intentional, but I wonder if there is another alternative to accomplishing this that I'm as yet unaware of. Thanks!

-------
Macbook Pro, Sonoma 14.5
Quire = 1.0.0-rc.15
node = v20.16.0

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @ewolfe1, Glad to hear you are experimenting with Quire, and thank you for reaching out. For this feature to work properly, all the images in an annotation set need to have the same aspect ratio. When using this feature in our Getty publications, we edit the images in Photoshop (adding blank spaces to the images as needed) to align them. Let us know if this helps!

**Reply by ewolfe1** [reply]
Thanks for the response. I figured that was the case, but good to have it confirmed. Perhaps having images of different aspect rations could be put onto a possible future version as an option? E.g., in the sample site, the Migrant Mother variants #2 displays stretched, which is another example use case where this could be needed. But I appreciate the feedback!
